lib/jekyll-feed.rb

```diff
@@ -12,6 +12,10 @@ def config
       @context.registers[:site].config
     end
 
+    def path
+      config["feed_path"] || "feed.xml"
```

Comment:
We'll need to use `Jekyll.sanitized_path(@site.source, config["feed_path"])` here. A malicious user could provide a relative path outside the Jekyll site source to expose the contents of files they otherwise wouldn't have access to. It may also be worth adding a test for this (e.g., verify a path of `../../foo` becomes `/foo`).